packages/core-api/src/versions/2/delegates/methods.ts

```diff
@@ -87,6 +97,9 @@ export function registerMethods(server) {
             ...request.query,
             ...paginate(request),
         }))
+        .method("v2.delegates.active", active, 30, request => ({
```

Comment:
The 30 seconds could probably be done as `blockTime * delegateCount` so it updates once a round.